mobius-rx/src/main/java/com/spotify/mobius/rx/Transformers.java

```diff
@@ -119,4 +122,57 @@ public void call(F f) {
       }
     };
   }
+
+  /**
+   * Creates an {@link Observable.Transformer} that will flatten the provided {@link Func1} into the
```

Comment:
This description sounds a little odd to me. "flatten the function"? IIUC, this transformer returns an observable that emits all the elements emitted by the input observable after applying the function to them. I guess the point of the flatmapping is to not invalidate the input observable if there is an error when applying the function? Is that the behaviour you'd want? If so, I think that should be made clearer in the docs.